docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
Please be more polite. And I'll just remove the date or do you think Debian sid is too shifting a point of reference and just shouldn't be added?